jcapi.go

```diff
@@ -240,26 +273,31 @@ func (jc JCAPI) extractStringArray(input []interface{}) []string {
 	return returnVal
 }
```

Comment:
Although, after having said that, you might also want to pre-allocate the return string array to the capacity of `len(input)`.  But I could see the argument to leave those to a different PR/refactor.